### DISCUSSION TITLE
Used by...

### DISCUSSION THREAD

**Discussion by raunofreiberg** [pacocoursey/cmdk#76] [General]
Would be interesting to hear and possibly list apps that have implemented the package.

**Comment by raunofreiberg** [comment]
## [Campsite](https://www.campsite.design/)

<img width="831" alt="CleanShot 2023-01-24 at 11 25 24@2x" src="https://user-images.githubusercontent.com/23662329/214206242-d2092b7e-101c-4f56-8817-de686a87fb52.png">

**Comment by raunofreiberg** [comment]
## [Mason](https://mason.app/)


https://user-images.githubusercontent.com/23662329/214206416-1c3f5434-4a22-45db-b03a-868fe8de87e6.mp4

**Comment by nkrekow** [comment]
we use it at [Wander](https://www.wander.com/index) but in a currently not public product (it will be later this year though!) :)

**Comment by dimuuu** [comment]
My side-project [subito.so](https://www.subito.so/) ✨ still WIP

**Comment by rishimohan** [comment]
## Using it in my app [Pika](https://pika.style)

![pika-1674550707800-2x](https://user-images.githubusercontent.com/14031295/214249954-9b513d22-bb38-4c62-add1-33ab66560ddf.png)
![pika-1674550695426-2x](https://user-images.githubusercontent.com/14031295/214249992-a2273a68-196c-4945-a379-309c3e45e6f6.png)

**Comment by ferrucc-io** [comment]
## [June](https://june.so)

<img width="1484" alt="Screenshot 2023-01-24 at 11 41 54" src="https://user-images.githubusercontent.com/8315559/214271218-ce487bb0-7fba-46b4-9280-5ba232b41c73.png">

**Comment by gorkemcetin** [comment]
[Magny](https://magny.io) (command palette as a service platform) uses Cmdk for its search bar.

<img width="1276" alt="magny dashboard" src="https://user-images.githubusercontent.com/1191477/214999651-660fd075-349e-405d-8223-2605b12085e4.png">

**Comment by raunofreiberg** [comment]
[Somehow](https://somehow.com/)

https://somehow.com/page/somehow/changelog/1685426952578-yg1upk
